apache-pulsar.yaml

```diff
@@ -0,0 +1,123 @@
+package:
+  name: apache-pulsar
+  version: 4.0.1
+  epoch: 0
+  description: Pulsar is a distributed pub-sub messaging platform with a very flexible messaging model and an intuitive client API.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-17-default-jvm
+      - openjdk-17-jre
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - maven
+      - openjdk-17-default-jdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/pulsar
+      tag: v${{package.version}}
+      expected-commit: 3ea527bf6b720dcfe074c4476257cce96ebc068a
+
+  - name: Build
+    runs: ./mvnw package -DskipTests
+
+  - name: Install
+    runs: |
+      dest=${{targets.contextdir}}/usr/share/java/pulsar
+      mkdir -p ${dest}
+
+      tar -zxf ./distribution/server/target/${{package.name}}-${{package.version}}-bin.tar.gz -C ${dest} --strip-components=1
+      rm -rf ${dest}/bin/*.cmd
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: ln -s /usr/share/java/pulsar ${{targets.contextdir}}/pulsar
+
+update:
+  enabled: true
+  github:
+    identifier: apache/pulsar
+    use-tag: true
+    strip-prefix: v
```

Comment:
There'll need to be a tag filter here to ignore things like candidate versions, e.g. https://github.com/apache/pulsar/releases/tag/v4.0.1-candidate-2